### ISSUE TITLE
Create resume.xsd to show user resume

### ISSUE THREAD

**Issue by paulodamaso** [zerocracy/datum#392]
Following implementation of https://github.com/zerocracy/farm/issues/1568 thread, we must create `pmo/resume.xsl)` file, which will show information about user resume when sent. After this implementation  remove ignore annotation from `TkJoinTest.showsResumeIfAlreadyApplied`

**Comment by 0crat** [comment]
@g4s8[/z](https://www.0crat.com/u/g4s8) please, pay attention to this issue

**Comment by g4s8** [comment]
@paulodamaso I don't think we need this file in datum actually, because it's derived data type from `resumes`. In datum, we keep only primary data schemes and `xsl`s, so it would be more correct to place `resume.xsl` in `src/main/resources/xsl` in farm repo, similar to `agenda.xsl` or `awards.xsl`.
